### PR TITLE
LPS-179147 Truncate fragment name

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by sandrodw3** [comment]
ci:test:sf

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/StructureTreeNode.js [hunk 1]
```diff
@@ -556,7 +561,9 @@ const NameLabel = React.forwardRef(
 						value={name}
 					/>
 				) : (
-					name || defaultName || Liferay.Language.get('element')
+					<span className="text-truncate">
+						{name || defaultName || Liferay.Language.get('element')}
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/StructureTreeNode.js:565]: Shouldn't we add a tooltip to see the full name of the fragment?

Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/StructureTreeNode.js:565]: The tree node itself already has it 😛

image

Comment (victorg1991) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/StructureTreeNode.js:565]: True, didn't notice that!

**Comment by victorg1991** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**